Tests/conf.json

```diff
@@ -2000,6 +2000,8 @@
         }
     ],
     "skipped_tests": {
+        "Test-Detonate URL - Crowdstrike": "Issue 19439",
```

Comment:
@ShahafBenYakir - https://github.com/demisto/etc/issues/19439 has been closed, are you sure it's related? Seems like a mistake in the issue number.

Reply:
Seems like a mistake indeed, you can run locally and if passed you can un-skip and see that it passes.